e2e/rules_cc/BUILD.bazel

```diff
@@ -44,3 +57,20 @@ platform_transition_binary(
     binary = ":supports_pic_",
     target_platform = "@toolchains_llvm_bootstrapped//platforms:linux_x86_64",
 )
+
+cc_test(
+    name = "windows_test",
+    srcs = ["windows_test.cc"],
+    # We run CI with `--config=stage1` so we will first compile a Windows-exec-flavored LLVM toolchain from source,
+    # then use that to build this target on the Windows host.
+    exec_compatible_with = [
+        "@platforms//os:windows",
+    ],
+    target_compatible_with = [
+        "@platforms//os:windows",
+    ],
+    linkopts = [
+        "-luuid",
```

Comment:
is this a test ? or actualy required ? I thought those were only for building clang ?

Reply:
`-luuid` is needed because the test uses the uuid lib :)
```
    GUID guid = GUID_NULL;
    (void)guid;
```